ekncontent/ekncontent/eknc-object-model.h

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2016 Endless Mobile, Inc. */
+
+#pragma once
+
+#include "eknc-content-object-model.h"
+
+G_BEGIN_DECLS
+
+EkncContentObjectModel *
+eknc_object_model_from_string (const gchar *str,
```

Comment:
Hmm, maybe I'll just move this to json node for now? And engine can just call `imports.gi.Json.from_string`, and then call the function.

Might be nice to expose a from_string function anyway, but I'll wait till engine/domain get ported to C before making any calls on API here. We don't need to expose anything function like this by then end of the port if we don't want to.